prow/cmd/tide/main.go

```diff
@@ -179,7 +179,7 @@ func main() {
 	server := &http.Server{Addr: ":" + strconv.Itoa(o.port)}
 
 	// Push metrics to the configured prometheus pushgateway endpoint or serve them
-	metrics.ExposeMetrics("tide", cfg().PushGateway)
+	metrics.ExposeMetrics("tide", cfg().PushGateway, nil)
```

Comment:
Let's prefer to add a new function `metrics.ExposeRegistry()` or something so that we don't need to change every single call like this

Reply:
One of my versions is like that.
My bad choice.